libs/dh/wholesale/feature-calculation-steps/src/lib/steps/25-production-per-gridarea.component.scss

```diff
@@ -33,7 +33,37 @@ watt-card h4 {
   margin: 0;
 }
 
+.sub-headline {
+  margin: 0;
+}
+
 hr {
   border: none;
   border-bottom: 1px solid var(--watt-color-neutral-grey-300);
 }
+
+// TODO: We don't have any spacing variable for 24px!
+dl {
+  border-top: 1px solid var(--watt-color-neutral-grey-300);
+  border-bottom: 1px solid var(--watt-color-neutral-grey-300);
+  margin: 24px 0 24px 0;
+  padding: 24px 0;
+  gap: 24px;
+
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+
+  dd {
+    margin: 0;
+  }
+}
+
+.new-row {
+  grid-column-start: 1;
+}
+
+
+// TODO: Typography should be updated in the design system
+.watt-headline-2, .watt-label {
```

Comment:
I will do a follow-up PR to update the typography styles in `Watt` and remove this.